src/Bicep.Cli/Commands/BuildParamsCommand.cs

```diff
@@ -63,14 +63,17 @@ public async Task<int> RunAsync(BuildParamsArguments args)
             if (bicepFileUri is not null &&
                 compilation.GetEntrypointSemanticModel().Root.TryGetBicepFileSemanticModelViaUsing().IsSuccess(out var usingModel))
             {
-                if (usingModel is not SemanticModel bicepSemanticModel)
```

Comment:
How does this work with Azure CLI if the user doesn't provide a template? E.g. [here](https://github.com/Azure/azure-cli/blob/8e3b15a02036a72f58cf94dac99d132dc7af57e7/src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py#L5453-L5470). Should this cause an error?

Reply:
I have the following content in `main.bicep` and `main.bicepparam` files;

```bicep
// main.bicep
param unusedParam int
```

```bicep
// main.bicepparam
using none
param unusedParam = 3
```

When I run the following command, it successfully build the files;

```bash
dotnet run --project src/Bicep.Cli/Bicep.Cli.csproj -- build-params ./main.bicepparam --bicep-file ./main.bicep --stdout
```

If I understand your comment correctly, you're asking what happens if we don't provide `--bicep-file ./main.bicep` option;

```bash
dotnet run --project src/Bicep.Cli/Bicep.Cli.csproj -- build-params ./main.bicepparam --stdout
```

I ran the command and saw it successfully build the `main.bicepparam` file.